go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/rickb777/date v1.12.5-0.20200422084442-6300e543c4d9
 	github.com/sergi/go-diff v1.2.0
 	github.com/tombuildsstuff/giovanni v0.20.0
-	github.com/tombuildsstuff/kermit v0.20230331.1120327
+	github.com/tombuildsstuff/kermit v0.20230412.1161912
```

Comment:
could we split the SDK update out into another PR?

Reply:
No, we can't split the SDK update only, because it introduces a behavioral breaking change, that the default build service won't be created by the service automatically.